### PR TITLE
Fixes and refactoring of classify_reported.py

### PR DESCRIPTION
**Pull request by tomasheiskanen**
- run from cmd line tool with --transcribe flag
- extracts audio from video to tempfile for whisper model transcription instead of specifying audio path
- move prompts to text files
- refactor to use same codebase instead of having duplicate code

### CODE REVIEW

File: tiktok_reporter_analysis/classify_reported.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+def extract_frames_n_audio(video_path, audio_path, num_frames=2):
+    clip = VideoFileClip(video_path)
+    audio = clip.audio
+    audio.write_audiofile(audio_path)
+    duration = clip.duration
+    frames = {}
+
+    for i in range(num_frames):
+        time = random.uniform(0, duration)
```
Comment (jmccrosky) [inline tiktok_reporter_analysis/classify_reported.py:20]: We should make sure we don't grab the same frame twice.  It also might be good to make sure the frames aren't too close together, but that could be an optimization for later if you prefer.

File: tiktok_reporter_analysis/__main__.py [hunk 2]
```diff
@@ -38,10 +42,14 @@ def analyze(video_path, frames_folder, checkpoint_path, results_path):
...
     )
     analyze_parser.add_argument("--results_path", help="path to the results folder", default="./data/results")
+    analyze_parser.add_argument("--testing", help="test with smaller random model", action="store_true")
+    analyze_parser.add_argument("--transcribe", help="transcribe audio and include in analysis", action="store_true")
 
     args = parser.parse_args()
 
     if args.command == "train":
         train(args.train_dir, args.labels_file, args.checkpoint_dir)
     elif args.command == "analyze":
```
Comment (jmccrosky) [inline tiktok_reporter_analysis/__main__.py:52]: I don't think the semantics here are quite right.  The difference between the code already checked in and what I'm adding here is that the former is a screen recording with (usually) multiple videos with scrolling and other events interleaved.  The new code is to analyze a scraped single TikTok video, which is just the TikTok video, with no UI overlay or anything else.

It happens that we're transcribing in the latter case, but not the former, but in theory we could also transcribe for the former case.

Make sense?

File: tiktok_reporter_analysis/classify_videos.py [hunk 3]
```diff
@@ -44,87 +43,10 @@ def classify_videos(frames_folder, results_path):
         one_third = min_frame + (max_frame - min_frame) // 3
         two_thirds = min_frame + 2 * (max_frame - min_frame) // 3
         # Find the frames that are closest to one third and two thirds of the way between min and max
```
Comment (jmccrosky) [inline tiktok_reporter_analysis/classify_videos.py:45]: If you want, you could use this approach in classify_reported.py and extract it to common.py.

**Review by jmccrosky — CHANGES_REQUESTED** [review]
Thanks for all the fixes!  Just a couple small suggestions.

File: tiktok_reporter_analysis/__main__.py (hunk 2 above)
Comment (tomasheiskanen) [reply tiktok_reporter_analysis/__main__.py]: Sure, did a new command for reported videos

**Review by jmccrosky — APPROVED** [review]
Looks great.  Thanks!

**Pull request merged**